src/modules/iotjs_module_buffer.c

```diff
@@ -411,12 +409,12 @@ JHANDLER_FUNCTION(Slice) {
   int length = end - start;
   IOTJS_ASSERT(length >= 0);
 
-  iotjs_jval_t jnew_buffer = iotjs_bufferwrap_create_buffer(length);
+  iotjs_jval_t jnew_buffer = iotjs_bufferwrap_create_buffer((size_t)length);
   iotjs_bufferwrap_t* new_buffer_wrap =
       iotjs_bufferwrap_from_jbuffer(&jnew_buffer);
   iotjs_bufferwrap_copy_internal(new_buffer_wrap,
-                                 iotjs_bufferwrap_buffer(buffer_wrap), start,
-                                 end, 0);
+                                 iotjs_bufferwrap_buffer(buffer_wrap),
+                                 (size_t)start, (size_t)end, 0);
```

Comment:
Like here, because both of `iotjs_bufferwrap_length` and `bound_range` returns `size_t`.
```c
 start += (int)iotjs_bufferwrap_length(buffer_wrap);
start = (int)bound_range(start, 0, iotjs_bufferwrap_length(buffer_wrap));
```
So my suggestion is like below. How do you think?
```c
int start = JHANDLER_GET_ARG(0, number);
size_t start_idx = start;

if (start < 0) {
  start_idx +=  iotjs_bufferwrap_length(buffer_wrap);
}
start_idx = bound_range(start_idx , 0, iotjs_bufferwrap_length(buffer_wrap));
```